nautobot/core/templates/search_form.html

```diff
@@ -1,5 +1,5 @@
 <div class="row" style="padding-bottom: 20px">
-    <div class="col-md-12 text-center">
+    <div class="col-md-12 col-sm-10 text-center pull-right">
```

Comment:
This seems to have broken the layout of the search form on the home page on mobile.

Before:

![image](https://github.com/nautobot/nautobot/assets/5603551/d0b01cd6-5aa4-4a7a-ad9d-b135c1e68f04)

After:

![image](https://github.com/nautobot/nautobot/assets/5603551/61000444-cfcf-406a-9621-4225f395ebde)

Reply:
this may be resolved with the auto collapse breakpoint on window size < 1007px

Reply:
confirmed that this is no longer an issue when navbar auto collapses.  Please have a look and let me know if the new behavior is acceptable for addressing this.